components/dash-core-components/tests/integration/misc/test_persistence.py

```diff
@@ -141,12 +141,12 @@ def make_output(*args):
 
     dash_dcc.find_element("#radioitems label:first-child input").click()  # red
 
-    range_slider = dash_dcc.find_element("#rangeslider")
-    dash_dcc.click_at_coord_fractions(range_slider, 0.5, 0.25)  # 5
-    dash_dcc.click_at_coord_fractions(range_slider, 0.8, 0.25)  # 8
+    # range_slider = dash_dcc.find_element("#rangeslider")
+    # dash_dcc.click_at_coord_fractions(range_slider, 0.5, 0.25)  # 5
+    # dash_dcc.click_at_coord_fractions(range_slider, 0.8, 0.25)  # 8
```

Comment:
Is this no longer working?

Reply:
It is activated back and functional now.